backend-ktor/src/main/kotlin/me/zwsmith/plugins/Models.kt

```diff
@@ -0,0 +1,22 @@
+package me.zwsmith.plugins
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Recipe(
+    val id: Int,
+    val name: String,
+    val description: String,
+    val quantifiedIngredients: List<QuantifiedIngredient>
+)
+@Serializable
+data class Ingredient(
+    val id: Int,
+    val name: String
+)
+@Serializable
+data class QuantifiedIngredient(
+    val ingredient: Ingredient,
+    val amount: Float,
+    val unit: String,
```

Comment:
Can we make the unit an enum? We can do that later too but just a thought

Reply:
Yea I was debating that too. Wasn't sure if that would complicate things more than I needed at this stage. But I'm down to make the adjustment 👍